rosidl_generator_c/test/test_interfaces.c

```diff
@@ -896,3 +941,325 @@ int test_submessages(void)
 
   return 0;
```

Comment:
I know this isn't in the diff, but could we change this to `return rc;` (so that we return an error code only when tests fail).